velero-csi-plugin/csi_snapshotter.go

```diff
@@ -34,6 +36,11 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
```

Comment:
nit: let's alias the `pkg/apis/velero/v1` package as `velerov1api` per our standard.

Reply:
The standard is documented at https://velero.io/docs/master/code-standards/#imports